packages/core/src/hosted-form/create-hosted-form-stored-card-service.ts

```diff
@@ -0,0 +1,17 @@
+import { createCheckoutStore } from '../checkout';
+
+import HostedFormFactory from './hosted-form-factory';
+import HostedFormStoredCardService from './hosted-form-stored-card-service';
+
+/**
+ * Creates an instance of `HostedFormStoredCardService`.
+ *
+ *
+ * @param host - Host url string parameter.
+ * @returns An instance of `HostedFormStoredCardService`.
+ */
+export default function createHostedFormStoredCardService(host: string) {
```

Comment:
```suggestion
export default function createStoredCardHostedFormService(host: string) {
```

I think it reads more naturally this way.